modules/docs/arrow-docs/docs/docs/optics/snoc/README.MD

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+title: Snoc
+permalink: /docs/optics/snoc/
+---
+
+## Snoc
+
+{:.beginner}
+beginner
+
+`Snoc` provides a [Prism]({{ '/docs/optics/prism' | relative_url }}) between `S` and its init `A` and last element `S`.
```

Comment:
`last element S. It provides a way to attach or detach elements on the end side of a structure.`